lib/src/clojure_lsp/feature/code_actions.clj

```diff
@@ -71,8 +71,12 @@
 
 (defn ^:private require-suggestion-actions
   [uri alias-suggestions]
-  (map (fn [{:keys [ns alias refer position]}]
-         {:title      (format "Add require '[%s %s %s]'" ns (if alias ":as" ":refer") (or alias (str "[" refer "]")))
+  (map (fn [{:keys [ns alias refer position count]}]
+         {:title      (format "Add require '[%s%s%s]'%s"
+                              ns
+                              (if alias (str " :as " alias) "")
+                              (if refer (str " :refer [" refer "]") "")
+                              (if count (str " × " count) ""))
```

Comment:
Awesome feature! maybe we should just be a little bit more explicit on what ` x 3` means for example, maybe change to `(3 usages)`? WDYT?
```
Add require '[clojure-lsp.feature.clean-ns :as f.clean-ns]' (3 usages)
```

Reply:
oh sorry, I started with that but it seemed too distracting. I think see how other people feel about it.